ledger/participant-integration-api/src/main/scala/platform/index/ReadOnlySqlLedgerWithMutableCache.scala

```diff
@@ -74,8 +74,7 @@ private[index] object ReadOnlySqlLedgerWithMutableCache {
     private def dispatcherOffsetSeqIdOwner(ledgerEnd: Offset, evtSeqId: Long) = {
       Dispatcher.owner(
         name = "transaction-log-updates",
-        zeroIndex =
-          (Offset.beforeBegin, 0L), // TODO: append-only: FIXME consolidating parameters table
+        zeroIndex = (Offset.beforeBegin, EventSequentialId.zero),
```

Comment:
Did you consider naming it `EventSequentialId.beforeBegin`? Two reasons:
* It's uniform wrt to its `Offset` counterpart
* It expresses the purpose not the implementation

What do you think?

Reply:
Renamed.